elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -0,0 +1,121 @@
+@import "../../pfe-sass/pfe-sass";
+$LOCAL: modal;
+
+$pfe-modal--breakpoint--medium: 640px;
+
+:host {
+	--pfe-modal--MaxWidth: 70vw;
+	--pfe-modal--MinWidth: 50vw;
+	--pfe-modal--MaxWidth--mobile: 94vw;
+	--pfe-modal--MaxHeight: 90vh;
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) 0 calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
+	--pfe-modal__overlay--BackgroundColor: #{pfe-color(overlay)};
+	--pfe-modal--CloseButtonSize: calc(#{pfe-var(ui--element--size)} - 4px);
+	display: block;
+	position: relative;
+}
+
+:host(:not([has_header])) {
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 4) calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
+}
+
+.pfe-modal {
+	&__outer {
+		display: flex;
+		position: fixed;
+		height: 100%;
+		width: 100%;
+		top: 0;
+		left: 0;
+		flex-direction: column;
+		align-items: center;
+		justify-content: center;
+		z-index: var(--pfe-z--modal, 2000);
+
+		&[hidden] {
+			display: none;
+			z-index: 0;
+		}
+	}
+	&__overlay {
+		position: fixed;
+		height: 100%;
+		width: 100%;
+		top: 0;
+		left: 0;
+		background-color: pfe-local($cssvar: BackgroundColor, $region: overlay);
+		cursor: pointer;
+
+		&[hidden] {
+			display: none;
+		}
+	}
+	&__window {
+		position: relative;
+		max-width: pfe-local(MaxWidth--mobile);
+		min-width: pfe-local(MinWidth);
+		margin: 0 auto;
+		box-shadow: pfe-var(box-shadow--lg);
+		background-color: pfe-color(surface--lightest);
+		border-radius: #{pfe-var(ui--border-radius)};
+
+		@media screen and (min-width: $pfe-modal--breakpoint--medium) {
+			max-width: pfe-local(MaxWidth);
+		}
+	}
+	&__container {
+		position: relative;
+		max-width: calc(100% - #{pfe-var(ui--element--size)});
+		padding: var(--pfe-modal--Padding);
+
+		&[hidden] {
+			display: none;
+		}
+
+		&[no_header] {
+			@media screen and (min-width: $pfe-modal--breakpoint--medium) {
+				padding: calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 6) calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);			
+			}
+
+			// Remove margin-top on the first slotted element that is not the header.
+			::slotted(*:nth-child(1)),
+			::slotted(*:nth-child(2)) {
+				margin-top: 0 !important;
+			}
+		}
+		::slotted([slot="pfe-modal--header"]) {
+			margin-top: 0 !important;
+			width: 90%;
+		}
+	}
+
+	&__content {
+		overflow-y: auto;
+		overscroll-behavior: contain;
+		max-height: pfe-local(MaxHeight);	
+		padding-right: calc(#{pfe-var(container-padding)} * 2);
+	}
+
+	&__close {
+		@include pfe-reset-button;
+		position: absolute;
+		top: calc(#{pfe-var(container-padding)} * .25);
```

Comment:
```suggestion
    top: 0;
    right: var(--pfe-theme--container-padding,1rem);
    background-color: white;
    border-radius: #{pfe-var(ui--border-radius)};
```

Reply:
I'm not sure if the background color or border-radius make sense for the close button - is this comment in the right place of the code?

Reply:
I think it's a reference to the `pfe-modal__window` 
We refactored the shadowdom a bit since this comment I think so it may not actually be referencing the same element any more.